modules/ssl/ssl_engine_vars.c

```diff
@@ -975,6 +979,62 @@ static const char *ssl_var_lookup_ssl_handshake_rtt(apr_pool_t *p, SSL *ssl)
     return NULL;
 }
 
+static const char *ssl_var_lookup_ssl_clienthello(apr_pool_t *p, const SSLConnRec *sslconn, const char *var)
+{
+#if OPENSSL_VERSION_NUMBER >= 0x10101000L
+    char *value;
+    modssl_clienthello_vars *clienthello_vars;
+    int i;
```

Comment:
`apr_size_t i` to match the type of `->extids_len`

Reply:
fixed